### PR TITLE
Refactor create_party.py

### PR DESCRIPTION
**Pull request by kimberlytze**
Clean up

@VOjha @vsiri

### CODE REVIEW

File: upnext/create_party.py [hunk 1]
```diff
@@ -58,15 +57,15 @@ def create_playlist(request, party_name):
...
-                party_id = party.uri.split(':')[-1]
+            username = 'up--next'
 
-                sp = spotipy.Spotify(auth=token_info['ACCESS_TOKEN'])
-                sp.user_playlist_add_tracks(username, party_id, [track_id])
+            tracks = party.track_set.order_by('-score', 'track_title', 'artist')
+            track_uris = map(lambda x: x.uri, tracks)
+
+            sp = spotipy.Spotify(auth=token_info['ACCESS_TOKEN'])
+            sp.user_playlist_add_tracks(username, party.uri, track_uris)
```
Comment (kimberlytze) [inline upnext/create_party.py]: Got rid of `party_id` and just used `party.uri`. Was there any particular reason we were doing the splitting before?

(`party.uri` is something like `spotify:user:up--next:playlist:501a0n8wr8u2GAzQpELW0L`,
while `party_id` is something like `501a0n8wr8u2GAzQpELW0L`)

Comment (VOjha) [inline upnext/create_party.py]: No idea. My guess is that I misunderstood how the spotipy endpoint works ¯_(ツ)_/¯

Comment (kimberlytze) [inline upnext/create_party.py]: Gonna get rid of it elsewhere then lol (where possible).

**Comment by VOjha** [comment]
Issue #84 

As long as things still work, this lgtm. Could you assign yourself to an issue before you do it though? :P

**Comment by kimberlytze** [comment]
Yep sorry :p

File: upnext/create_party.py (hunk 1 above)
Comment (kimberlytze) [inline upnext/create_party.py]: Weird it says it takes id, but uri still works. Gonna switch it back in case LOL.

**Comment by vsiri** [comment]
LGTM

**Pull request closed without merging**